### DISCUSSION TITLE
Performance degradation from 464 to 465

### DISCUSSION THREAD

**Discussion by C-h-e-r-r-y** [trinodb/trino#24240] [General]
i ahve updated trino from 464 to 465 and got a lot of erros that worker is not reachable. I run about 170+ queries and about 50+ of them fails. But for 464 version there are no fails.

Ideally I should create en issue but I can not provide any data or share queries. So just asking whether somebody tried and faced same issue.

**Comment by losipiuk** [comment]
Can you share the nature of failures? Exact exception you get when you run using `trino-cli --debug`
Are the workers crashing?

**Comment by C-h-e-r-r-y** [comment]
Increasing all `max-requests-queued-per-destination` up to 3072 fixed issue. 465 works but slower than 464 :-(
